autogen/main/cluster.tf.tmpl

```diff
@@ -677,12 +680,12 @@ resource "google_container_cluster" "primary" {
       gcfs_config {
```

Comment:
I believe that `gcfs_config` is still beta, so will need to still gate this block.

Reply:
Yeah was going to rebase after #2098 originally, but now they may need to go together?

Reply:
My quick read is that even in [v6.3](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster) while `node_config_defaults` is GA, `node_config_defaults.gcfs_config` is still beta?

Reply:
correct. The narrower fix I ended up with there still keeps `gcfs_config` gated (however breaks because then `node_config_defaults` is empty, which I think will be a potential issue either way unless we do the dynamic block?)

Reply:
Hi @wyardley - Looks like `gcfs_config ` was promoted to GA in [v6.5.0](https://github.com/hashicorp/terraform-provider-google/releases/tag/v6.5.0), so in the subsequent breaking release (35?) this can be resolved.